third_party/cuda/BUILD

```diff
@@ -9,6 +9,21 @@ config_setting(
     ],
 )
 
+cc_library(
```

Comment:
If this comes in a different `http_archive` or `local_repository`, it should have its own BUILD file